src/query/storages/parquet/tests/it/merge_io.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use bytes::Bytes;
+use common_base::base::tokio;
+use common_storages_parquet::InMemoryRowGroup;
+use opendal::services::Memory;
+use opendal::Operator;
+use parquet::basic::ConvertedType;
+use parquet::basic::Repetition;
+use parquet::basic::Type as PhysicalType;
+use parquet::file::metadata::RowGroupMetaData;
+use parquet::schema::types::*;
+
+#[tokio::test]
+async fn test_merge() {
```

Comment:
Better now.

But this unit test looks doing a simple check, we can make it more robust, here some suggestions:
1. generate the parquet with random data & random column
2. check the random ranges as expected

Reply:
`------------
     Summary [ 183.557s] 1452 tests run: 1451 passed (1 slow), 1 failed, 9 skipped
        FAIL [   0.037s] common-base::it runtime::test_shutdown_long_run_runtime
error: test run failed
Error: Process completed with exit code 100.
`

Reply:
```
------------
     Summary [ 183.557s] 1452 tests run: 1451 passed (1 slow), 1 failed, 9 skipped
        FAIL [   0.037s] common-base::it runtime::test_shutdown_long_run_runtime
error: test run failed
Error: Process completed with exit code 100.
```